MSAL/src/public/MSALAccount.h

```diff
@@ -46,5 +48,10 @@
  */
 @property (readonly, nonnull) NSString *environment;
 
+/*!
+ Array of all tenants which are logged in for this account.
```

Comment:
nit: logged in is an ambiguous term. I'd rather say something like "all tenants for which a token has been requested by the client"

Reply:
:+1: